_gsocproposals/2018/proposal_DD4hepROOT.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Tessellated geometry support for DD4hep and ROOT/TGeo
+layout: gsoc_proposal
+project:
+- DD4hep
+- ROOT
+year: 2018
+organisation: CERN
+---
+
+## Description
+For the success of an experiment it is important to provide a consistent detector description to simulation, reconstruction and analysis applications from a single source. [DD4hep](https://dd4hep.cern.ch) is a framework for the complete description of detector models based on a single source of information.  When referring to the detector description this includes, in addition to the geometry and the materials used in the device, also parameters describing e.g. the detection techniques, constants required for alignment and calibration, description of the readout structures and conditions data. The main motivation behind the [DD4hep](https://dd4hep.cern.ch) framework is to devise a toolkit that addresses all these issues for all the stages of an experiment. The tool relies on usage of pre-existing and widely used software, combining it into a consistent generic detector description. The main components are the [ROOT](https://root.cern.ch/) geometry package (TGeo), which is used for construction and visualisation of the geometry, and the [Geant4](http://geant4.cern.ch/) simulation toolkit, which can be interfaced via [DD4hep](https://dd4hep.cern.ch) to perform detector simulation in complex detector designs.
```

Comment:
for DD4hep URL, use `http` instead of `https` as the certificate used is non standard and makes the test to fail.